src/watch.ts

```diff
@@ -129,29 +166,25 @@ export class Watch {
       return;
     }
 
-    try {
-      if (change.deleted) {
-        // Delete package directly in index
-        // Filter does not support async/await but there is no concurrency issue with this
-        throw new Error('deleted');
-      }
-      const res = await npm.getDoc(change.id, change.changes[0].rev);
+    if (job.retry > 0) {
+      // retry backoff
+      const backoff = Math.pow(job.retry + 1, config.retryBackoffPow) * 1000;
+      log.info('Retrying (', job.retry, '), waiting for', backoff);
+      await wait(backoff);
```

Comment:
does this slow down the processing of other items, or is it just this one promise?

Reply:
As mentioned in the comments above, it is in fact blocking because:
- if it's third party API failure, this is going to impact all others packages so no need to rush anything
- if it's only related to this package, at worst we have waited 1minutes.

Since we are have not achieved proper real time, I feel like it's okay. Wdyt?